src/core/trtserver.h

```diff
@@ -376,6 +377,96 @@ TRTSERVER_EXPORT TRTSERVER_Error* TRTSERVER_TraceNew(
 /// \return a TRTSERVER_Error indicating success or failure.
 TRTSERVER_EXPORT TRTSERVER_Error* TRTSERVER_TraceDelete(TRTSERVER_Trace* trace);
 
+/// Get the activity userp of the trace object, which is supplied in the call to
+/// TRTSERVER_TraceNew.
+/// \param trace The trace object.
+/// \param userp Returns the activity userp of the trace object.
+/// \return a TRTSERVER_Error indicating success or failure.
+TRTSERVER_EXPORT TRTSERVER_Error* TRTSERVER_TraceUserp(
+    TRTSERVER_Trace* trace, void** userp);
+
+/// Get the name of the model being traced. The caller
+/// does not own the returned string and must not modify or delete
+/// it. The lifetime of the returned string extends only as long as
+/// 'trace' and must not be accessed once 'trace' is deleted.
+/// 'model_name' is guaranteed to be valid at invocation of
+/// TRTSERVER_TraceManagerReleaseTraceFn_t
+/// \param trace The trace object.
+/// \param model_name Returns the name of the model being traced.
+/// \return a TRTSERVER_Error indicating success or failure.
+TRTSERVER_EXPORT TRTSERVER_Error* TRTSERVER_TraceModelName(
+    TRTSERVER_Trace* trace, const char** model_name);
+
+/// Get the version of the model being traced.
+/// 'model_version' is guaranteed to be valid at invocation of
+/// TRTSERVER_TraceManagerReleaseTraceFn_t
+/// \param trace The trace object.
+/// \param model_version Returns the version of the model being traced.
+/// \return a TRTSERVER_Error indicating success or failure.
+TRTSERVER_EXPORT TRTSERVER_Error* TRTSERVER_TraceModelVersion(
+    TRTSERVER_Trace* trace, int64_t* model_version);
+
+/// Get the id of the trace object.
```

Comment:
Add "Each trace object created during execution of the inference server has a unique id."